app/models/task.py

```diff
@@ -1,6 +1,48 @@
 from flask import current_app
 from app import db
+from flask import Blueprint
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String(50))
+    description = db.Column(db.String(50)) 
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey("goal.id"), nullable=True)
+    
+
+    def create_json(self):
+        if self.completed_at == None:
+            completed = False
+        else:
+            completed = True
+
+        return {
+                'id' : self.id,
+                'title' : self.title,
+                'description' : self.description,
+                'is_complete' : completed
+        }
+
+    def create_json_with_goal_id(self):
```

Comment:
You could make this helper a part of the above `create_json` function using an `if` statement to include `goal` if the goal field exists.